unfoldingWord/translationCore
# Added Methods To Handle Outside Selection For Highlight

#### This pull request addresses:

Write a few sentences about what your pull request addresses, the relevant issue numbers, etc.
This PR address the issue with highlighting text and landing outside the intended container.

#### How to test this pull request:
Include testing instructions so that other developers know what to look for, and how to make sure that your feature/bugfix/enhancement really works.
To test this PR try highlighting and ending outside of the display container, try with both tools
![bitmoji](https://render.bitstrips.com/v2/cpanel/10216108-249627154_2-s1-v1.png?transparent=1&palette=1&width=246)


---
This change is [<img src="https://reviewable.io/review_button.svg" height="34" align="absmiddle" alt="Reviewable"/>](https://reviewable.io/reviews/unfoldingword-dev/translationcore/835)
